client/components/account-balances/balances-tab-panel.tsx

```diff
@@ -10,6 +10,7 @@ import { Flex, TabPanel } from '@wordpress/components';
 import { useAllDepositsOverviews } from 'wcpay/data';
 import { getCurrencyTabTitle } from './utils';
 import BalanceBlock from './balance-block';
+import { AccountOverview } from 'wcpay/types/account-overview';
```

Comment:
Thanks for pointing this out. This is due to the `export` keyword I added to `export declare namespace AccountOverview`.

I realise I don't have a strong reason for this change aside from a personal preference.

I've reverted to the existing behaviour (global type/namespace definition rather than exported) in 959ca40.

Reply:
TIL.